README.md

```diff
@@ -25,6 +25,15 @@
 
 Roslyn provides open-source C# and Visual Basic compilers with rich code analysis APIs.  It enables building code analysis tools with the same APIs that are used by Visual Studio.
 
+### Language Design Discussion Moved
```

Comment:
💭 The word 'Moved' is probably unnecessary here, and over time it seems like it will make more sense to have it not included.